src/test/java/uk/gov/hmcts/reform/sendletter/tasks/reports/EmailSenderTest.java

```diff
@@ -0,0 +1,132 @@
+package uk.gov.hmcts.reform.sendletter.tasks.reports;
+
+import com.icegreen.greenmail.util.ServerSetupTest;
+import org.apache.commons.mail.util.MimeMessageParser;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.RegisterExtension;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import uk.gov.hmcts.reform.sendletter.jupiter.GreenMailExtension;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Properties;
+import javax.activation.DataSource;
+import javax.mail.Address;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class EmailSenderTest {
+
+    private static final String TEST_LOGIN = "test@localhost.com";
+    private static final String TEST_PASSWORD = "test_password";
+
+    private static final String SUBJECT = "email subject";
+    private static final String RECIPIENT_A = "recipient A <a@localhost>";
+    private static final String RECIPIENT_B = "b@localhost";
+    private static final File ATTACHMENT_FILE;
+    private static final Attachment ATTACHMENT_1;
+    private static final Attachment ATTACHMENT_2;
+
+    static {
+        try {
+            ATTACHMENT_FILE = File.createTempFile("unit-test", "tst");
```

Comment:
can we not have a file in resources folder for test instead of creating temp file for each run >